src/main/target/common_post.h

```diff
@@ -280,6 +280,10 @@
 #undef USE_ACRO_TRAINER
 #endif
 
+#if (!defined(USE_GPS_RESCUE) || !defined(USE_CMS_FAILSAFE_MENU) )
```

Comment:
Stray whitespace. :space_invader:

Reply:
@mikeller  should i swash  this after i fix stray ..
i also need to test it again ... after changes for the move of the menu  :(